web/sites/default/settings.php

```diff
@@ -591,7 +591,7 @@
  * If you don't wish to disable FLoC in Chrome, you can set this value
  * to FALSE.
  */
-# $settings['block_interest_cohort'] = TRUE;
+ #$settings['block_interest_cohort'] = FALSE;
```

Comment:
unnecessary changes. You should avoid the "git add ." command and it is better to use "git add -p" instead.